docs/insomnia/get-started/mock-server/allowlist.md

```diff
@@ -3,60 +3,29 @@ layout: article-detail
 title:  Allowlist for Insomnia
 ---
 
-<!-- for docs rework: revise this page like we normally would -->
+For enterprise users utilizing Insomnia, it's crucial to ensure that specific domains are allowlisted in your network. This step is essential to guarantee that all features of Insomnia, including updates, synchronization, documentation, and necessary backend services work without any hindrance in environments with restricted internet access.
 
-## Introduction
+If you have issues after allowlisting Insomnia hostnames, contact your IT support or reach out to Insomnia's customer service at <support@insomnia.rest> or <https://support.konghq.com/support/s/>.
 
-For enterprise users utilizing Insomnia, it's crucial to ensure that specific domains are whitelisted in your network. This step is essential to guarantee that all features of Insomnia, including updates, synchronization, and documentation, work without any hindrance in environments with restricted internet access.
+## Hostnames to allowlist
 
-## Domains to Whitelist
+Ensure the following domains are allowlisted to use Insomnia:
 
-To fully utilize Insomnia, please ensure the following domains are whitelisted:
+| Hostname | Description |
```

Comment:
@fabianrbz I added a table here and a note below like we have in our docs repo, but they aren't quite rendering correctly. I probably don't have to use the note format, I could change it to something else if it's too difficult to add that formatting in this repo. But I'd like to keep the table. Is the table formatting something you could add?

Reply:
@cloudjumpercat It's rendering an HTML table, the problem is that it doesn't have any styles applied,
I've pushed a commit to this branch that applies Bootstrap's default table styles to all the tables. Hopefully, that's enough (we can always tweak those).

Let's keep track of all the style differences.

Reply:
@fabianrbz Thanks!

> Let's keep track of all the style differences.

Where would be the best place to keep track of the differences?

Reply:
Jira?

Reply:
@fabianrbz I started documenting the differences in the styling Jira ticket: https://konghq.atlassian.net/browse/DOCU-3897. The table looks good for this doc, so we can just leave the default styling for now in this PR and if you want to make any adjustments in another PR/the styling PR, that would work!